pages/content/shared/data/roadshow.yaml

```diff
@@ -0,0 +1,156 @@
+nextstops:
```

Comment:
There was an update on the `production` branch for the stations, can you check if it still matches those here?

Reply:
are the added fields to nextstop unwanted?